src/towncrier/newsfragments/275.feature

```diff
@@ -0,0 +1 @@
+Users can now write the content in the ``create`` subcommand.
```

Comment:
I know I already had you change it to this but I think it could be a bit more explicit in some way.  Maybe:

```suggestion
The new ``--edit`` option of the ``create`` subcommand launches an editor for entering the contents of the newsfragment.
```